routes/data.js

```diff
@@ -136,3 +188,8 @@ exports.organizationsPost = async function (req, res) {
     .update(updatedOrganizationData);
   res.sendStatus(201);
 };
+
+exports.categoriesGet = async function (req, res) {
```

Comment:
Is it necessary to have ```getCategories``` as well as ```categoriesGet```? I'm not sure what the difference is supposed to be...

Reply:
Changed to only use `getCategories` 👍 

There were two methods originally because Dan needed it as a GET response while I needed it to return an array, but updated it to draw from the same method.